service/dap/server_test.go

```diff
@@ -333,32 +335,19 @@ func TestSetBreakpoint(t *testing.T) {
 		}
 
 		client.ScopesRequest(1000)
-		scResp := client.ExpectScopesResponse(t)
-		if len(scResp.Body.Scopes) != 2 {
-			t.Errorf("\ngot  %#v\nwant len(Scopes)=2", scResp.Body.Scopes)
-		} else {
-			expectScope := func(got dap.Scope, name string, varRef int) {
-				t.Helper()
-				if got.Name != name || got.VariablesReference != varRef || got.Expensive {
-					t.Errorf("\ngot  %#v\nwant Name=%q VariablesReference=%d Expensive=false", got, name, varRef)
-				}
-			}
-			expectScope(scResp.Body.Scopes[0], "Arguments", 1000)
-			expectScope(scResp.Body.Scopes[1], "Locals", 1001)
-		}
+		scopes := client.ExpectScopesResponse(t)
+		expectScope(t, scopes, 0, "Arguments", 1000)
```

Comment:
Two separate scopes are an improvement on vscode-go, which mixes them into one.
<img src="https://user-images.githubusercontent.com/51177946/88310647-c50f6c00-ccc4-11ea-8ae1-bbfdf21a7b47.png" alt="alt text" width="250">
This is more consistent with delve which has separate requests for each.
<img src="https://user-images.githubusercontent.com/51177946/88310723-dbb5c300-ccc4-11ea-9da4-4882a117f422.png" alt="alt text" width="300">